### DISCUSSION TITLE
RaspberryPi 5 Cooling

### DISCUSSION THREAD

**Discussion by neversleep404** [raspiblitz/raspiblitz#4278] [Show and tell]
I'm running the raspiblitz-arm64-rpi-image-2023-11-23-9f23886 on a setup that includes:

Pi 5 with 8GB RAM
Raspberry Pi Active Cooler
A 3.5" RPi Display with GPIO connection and an XPT2046 Touch Controller
2TB Samsung SSD (originally from my RaspiBlitz v1.10.0 Pi 4 8GB setup)
Core Lightning Node
CLBOSS
The system operates stably at a 92% CPU load with a temperature of 58°C. However, I noticed that the back of the board was getting hot. To address this, I attached the bottom plate from the BerryBase Armor case designed for Raspberry Pi 4 to the Pi 5 using M2.5 Series Hex Brass Spacers. This modification has significantly cooled the back side of the board.

![cool - 1](https://github.com/raspiblitz/raspiblitz/assets/80689652/61c89ecc-d47b-40fd-9eeb-ae8a400ee42d)

**Comment by rootzoll** [comment]
That is a cool idea to reuse the old RPi4 cooling case. Thx for sharing.

**Comment by neversleep404** [comment]
I've been considering how the RaspiBlitz can accommodate a PCIe HatDrive in the future while also ensuring efficient cooling. What I like about the original case is that the cooling fan extracts air, thereby cooling the entire board. I've also added extra heat sinks to the key chips.

![bl1](https://github.com/raspiblitz/raspiblitz/assets/80689652/2d6ff5eb-5672-4126-b267-3ccf5476dcd8)

I rotated the display by 90 degrees to allow the addition of extra GPIO modules or, in my case, a PCIe HatDrive. Beneath the Raspberry Pi, there was enough space for a massive heat sink. I'm hoping this will keep the SD card cooler. Without cooling, the SD card in my RaspiBlitz gets very warm.

![b2](https://github.com/raspiblitz/raspiblitz/assets/80689652/69e417d5-bc9e-4c8e-b9e5-498857d6a44d)

I plan to manage the cable clutter with a cable duct. In any case, the RaspiBlitz has now become a neat desk gadget that takes up minimal space.
And yes, it's overengineered, but that's part of the fun.

**Comment by kravens** [comment]
![image](https://github.com/raspiblitz/raspiblitz/assets/9646902/a08934b5-d301-4ed6-a796-518cd85a0fa8)
I've got the NVME bottom hat, that supports a 80mm NVME SSD below the Pi5.
Still need to buy a cheap 2TB SSD and test it out with the next Raspiblitz 1.11 image.

**Comment by neversleep404** [comment]
> I've got the NVME bottom hat, that supports a 80mm NVME SSD below the Pi5.

Excellent! How do you access the SD card when it's installed? Additionally, could you share your experiences regarding the SSD's performance and its operational temperature?

**Reply by kravens** [reply]
I think my goal is to go SD-cardless with this Pi, the Pi5 has a higher USB-power delivery limit than the Pi4 so until now my Samsung T7 portable SSD is running perfectly on the Pi5 (it does not run at all on a Pi4).

**Comment by neversleep404** [comment]
I ordered an NVMe Base from Pimoroni. 
I think it will be a chic, slim piece of hardware, with only two cables: power and network.

![P5](https://github.com/raspiblitz/raspiblitz/assets/80689652/57d16667-1565-42cc-b510-e72a1fd29148)


A 3.5" RPi Display 
Raspberry Pi Active Cooler
Pi 5 with 8GB RAM
and the NVMe Base.

**Comment by neversleep404** [comment]
I received a Pineberry Pi HatDrive! The NVMe overheated while copying the blockchain from a USB SSD and subsequently slowed down. Therefore, I've now attached copper heat sinks and can additionally cool the NVMe with an external fan for such mega copying tasks in the future. 

![piHat2](https://github.com/raspiblitz/raspiblitz/assets/80689652/2b3e9aa1-4b95-42be-abd3-b49ef143b766)

Nonetheless, everything has been running stable in PCIe Gen 3 mode so far, although I don't dare to use this Pi 5 productively (i.e., with Bitcoins in the wallet).


![PiHat1](https://github.com/raspiblitz/raspiblitz/assets/80689652/8961088c-9d23-4b32-8de2-8b2f056abfea)

**Comment by rootzoll** [comment]
>The NVMe overheated while copying the blockchain from a USB SSD
@neversleep404 yeah the bottom NVMe board might be the cooler location .. thanks for the report

**Comment by NilByte** [comment]
My experience with Pi5 so far is that passive cooling will be fine.

I'm using ED-Pi5Case-OS / OB.

https://www.tomshardware.com/raspberry-pi/raspberry-pi-cases/edatec-raspberry-pi-5-cases-review-passively-cool

https://www.edatec.cn/uploads/soft/231225/Pi5OPENCASE-en.pdf

Ambient around 20°C, Pi5 around 50°C at normal load, so delta t around 30°C at normal load. Cost around $10.

![image](https://github.com/user-attachments/assets/e31a9264-1fc8-4024-be6e-8dc9cf4a6e0a)
